src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -52,7 +51,6 @@ private void CleanUpPersistentFolder()
         {
         }
 
-        [Fact]
```

Comment:
they don't run in xunit.runner.wpf.  I'm not sure about th ecommand line runner.